common-app-lib/src/cmr/common_app/static.clj

```diff
@@ -184,6 +183,15 @@
        (md->html)
        (selmer/render {})))
 
+(defn parse-out-env [cmr-host]
+  (let [regex-pattern #"cmr\.(.*?)\.earthdata\.nasa\.gov"
```

Comment:
Although not part of our concerns, verify with NOAA what their URL is as this may break their functionality.

Reply:
we're using a deconfig now